ajax/libs/tui-calendar/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "tui-calendar",
+  "filename": "tui-calendar.min.js",
+  "description": "A full featured JavaScript schedule calendar.",
```

Comment:
Let's follow the upstream: https://github.com/nhnent/tui.calendar/blob/master/package.json#L7
> TOAST UI Calendar